### PR TITLE
remove unused deps

### PR DESCRIPTION
**Pull request by ocefpaf**
@callumrollo also using this one to test the CI failure you are experimenting in #192.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Setup Conda
-      uses: s-weigand/setup-conda@v1
+      uses: s-weigand/setup-conda@v1.0.5
```
Comment (ocefpaf) [inline .github/workflows/tests.yml:20]: @callumrollo looks like it is the latest release of this GHA (1.0,6). Pinning to the previous one fixes it. I'll push that change to your branch.

**Comment by callumrollo** [comment]
@ocefpaf can we merge this? Then I can update the two other PRs

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -17,22 +17,28 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Setup Conda
-      uses: s-weigand/setup-conda@v1
+      #uses: s-weigand/setup-conda@v1.0.5
+      uses: s-weigand/setup-conda@fix-env-activation
```
Comment (s-weigand) [inline .github/workflows/tests.yml:21]:
```suggestion
      uses: s-weigand/setup-conda@fix-env-activation-dist
```
If this fixes the issue with the activation of the `TEST` env.
I will release a new version of `setup-conda` which should fix the usage of the `v1` tag.
Thanks for testing ❤️ 
Ref:
https://github.com/s-weigand/setup-conda/issues/125

Comment (ocefpaf) [reply .github/workflows/tests.yml]: Sorry for the missing `-dist` there. I tried that using the mobile web interface during the weekend, it was doomed to fail ;-p

Thanks for the quick fix. Let's see how the CIs go.

Comment (ocefpaf) [reply .github/workflows/tests.yml]: @s-weigand  works like a charm! Thanks!! I'll wait for the release to merge this one.

**Comment by ocefpaf** [comment]
> @ocefpaf can we merge this? Then I can update the two other PRs

Not right now. Let's wait for an official release of the GHA. Meanwhile I'm OK downgrading to the previous version.

**Comment by s-weigand** [comment]
Give me 5min and you will have the release 😄 
Thanks again for testing!

**Comment by ocefpaf** [comment]
Thanks so much for the quick fix @s-weigand!

**Pull request merged**